paseos/communication/is_in_line_of_sight.py

```diff
@@ -57,3 +88,133 @@ def is_in_line_of_sight(actor, other_actor, epoch: pk.epoch, plot=False):
             )
         return True
     return False
+
+
+def _is_in_line_of_sight_ground_station_to_spacecraft(
+    ground_station,
+    spacecraft,
+    epoch: pk.epoch,
+    minimum_altitude_angle: float,
+    plot=False,
+):
+    """Determines whether a position is in line of sight of this actor
+
+    Args:
+        ground_station (GroundstationActor): The actors to check line of sight from
+        spacecraft (SpacecraftActor): The actor to check line of sight with
+        epoch (pk,.epoch): Epoch at which to check the line of sight
+        minimum_altitude_angle(float): The altitude angle (in degree) at which the actor
+        has to be in relation to the surface, has to be between 0 and 90.
+        to be visible from this ground station. Has to be > 0 and < 90.
+        Only relevant if one of the actors is a ground station.
+        plot (bool): Whether to plot a diagram illustrating the positions.
+
+    Returns:
+        bool: true if in line-of-sight.
+    """
+    assert (
+        minimum_altitude_angle < 90 and minimum_altitude_angle > 0
```

Comment:
Why not -90 < angle < 90?

Reply:
Everything < 0 means under the horizon in Skyfield https://rhodesmill.org/skyfield/positions.html#azimuth-and-altitude-from-a-geographic-position

Reply:
Well, if you are below the horizon you cannot communicate?

Reply:
Yes, but why don't you return False instead of throwing an error?

Reply:
!? I think you misunderstand. This is a user define parameter specifying when the satellite is visible. E.g. you give it 30° then they are in contact while the satellite is between 30° and 90°. The assert is to make sure the user understood the value correct not to check for line of sight. If you give it 0° and the satellite is under horizon it will just return False.